docs/ANONYMITY_NETWORKS.md

```diff
@@ -1,149 +1,176 @@
 
-Receiving anonymity connections is done through the option
-`--anonymous-inbound`. This option tells `monerod` the inbound address, network
-type, and max connections:
 
+#### Peer handling
+
+If desired, outgoing peers can be specified manually:
 ```
---anonymous-inbound rveahdfho7wo4b2m.onion:28083,127.0.0.1:28083,25
---anonymous-inbound cmeua5767mz2q5jsaelk2rxhf67agrwuetaso5dzbenyzwlbkg2q.b32.i2p:5000,127.0.0.1:30000
+--add-peer 136.XXX.121.74
+--add-peer rveahdfho.onion:18083
+--add-peer cmeua5767.b32.i2p:5000
+
+--add-exclusive-node 136.XXX.121.74
+--add-exclusive-node rveahdfho.onion:18083
+--add-exclusive-node cmeua5767.b32.i2p:5000
 ```
+The option can be repeated and recognizes Tor, I2P, and IPv4 addresses. If you add a hidden service peer, the option `--tx-proxy` is mandatory. Using `--add-exclusive-node` will prevent the usage of seed nodes on _all_ networks, which will typically be undesirable.
 
-which tells `monerod` that a max of 25 inbound Tor connections are being
-received at address "rveahdfho7wo4b2m.onion:28083" and forwarded to `monerod`
-localhost port 28083, and a default max I2P connections are being received at
-address "cmeua5767mz2q5jsaelk2rxhf67agrwuetaso5dzbenyzwlbkg2q.b32.i2p:5000" and
-forwarded to `monerod` localhost port 30000.
-These addresses will be shared with outgoing peers, over the same network type,
-otherwise the peer will not be notified of the peer address by the proxy.
 
-### Wallet RPC
+## Inbound connections
 
-An anonymity network can be configured to forward incoming connections to a
-`monerod` RPC port - which is independent from the configuration for incoming
-P2P anonymity connections. The anonymity network (Tor/i2p) is
-[configured in the same manner](#configuration), except the localhost port
-must be the RPC port (typically 18081 for mainnet) instead of the p2p port:
+An anonymity network can be configured to forward incoming connections to `monerod`. Add the option
+```
+--anonymous-inbound rveahdfho.onion:18083,127.0.0.1:18083,25
+--anonymous-inbound cmeua5767.b32.i2p:5000,127.0.0.1:30000
+```
+which tells `monerod` that a max of 25 inbound Tor connections are being
+received at address `rveahdfho.onion:18083` and forwarded to `monerod`
+localhost port `18083`.  
+In this case for I2P, the default max. connections are being received at address `cmeua5767.b32.i2p:5000` and forwarded to `monerod` localhost port `30000`. The connecting hidden service address will be seeded to other peers on the AN.
 
+Incoming connections require a hidden service. Regards Tor, add
 ```
+# file /etc/tor/torrc
 HiddenServiceDir /var/lib/tor/data/monero
-HiddenServicePort 18081 127.0.0.1:18081
+# HiddenServicePort 18081 127.0.0.1:18081 # rpc port
+HiddenServicePort 18083 127.0.0.1:18083   # tx port
 ```
+and restart `sudo systemctl restart tor.service`. This will generate your `.onion` address in `/var/lib/tor/data/monero/hostname`. 
```

Comment:
Mention that the onion address from this file must be listed in the `monerod` configuration to enable inbound connections. Otherwise `monerod` doesn't know the address to advertise to peers.